docs/reference/aggregations/bucket/histogram-aggregation.asciidoc

```diff
@@ -175,6 +180,13 @@ POST /sales/_search?size=0
 --------------------------------------------------
 // TEST[setup:sales]
 
+When aggregating ranges, it is also possible to get buckets outside of your query. If the range is included in the
```

Comment:
I think the last sentence of this para is pretty important. I'd bring it to the front and reword. Something like this:

```
When aggregating based on range values,
buckets are based on the values of returned documents.
This means the response may include buckets outside of a query's range.

For example, 
a query returns documents containing a range value greater than `100`
and uses an interval of `50`.
A returned document contains a range value of `50` to `150`.
Because buckets are based on document values rather than the query,
the response counts this document in three buckets:
`50`, `100`, and `150`.
```